convert_osm/src/reader.rs

```diff
@@ -116,7 +116,7 @@ pub fn read(path: &str, input_gps_bounds: &GPSBounds, timer: &mut Timer) -> Resu
                     }
                 }
                 if !nodes.is_empty() {
-                    doc.ways.insert(id, Way { tags, nodes, pts });
+                    doc.ways.insert(id, Way { nodes, pts, tags });
```

Comment:
Whoa, that's an awesome lint! I always want to sync up the order of fields, but it's so much work to spot differences manually

Reply:
Yeah, it's especially nice with rust-analyzer providing automated fixes. 😄